HW4/task1.py

```diff
@@ -0,0 +1,16 @@
+some_text = input("Write here some text: ")
+
+
+if some_text.isdigit():
+    some_text = int(some_text)
+    print("This is a number")
+    if(some_text % 2 == 0):
```

Comment:
nit: brackets could be omitted here
```suggestion
    if some_text % 2 == 0:
```